docs/pages/reference/signature-algorithms.mdx

```diff
@@ -215,3 +215,56 @@ The signature algorithm suites are designed to simplify the configuration burden
 We did not want to expose 100 configuration knobs to modify every single
 signature Teleport does, which could lead to thousands of possible combinations
 we'd have to support, and could create the possibility for insecure combinations.
+
+### What if my HSM doesn't support ECDSA keys?
+
+If you are using a PKCS#11 HSM that does not support ECDSA keys, we recommend
+you continue to use the `legacy` algorithm suite.
+For an existing cluster it's possible you could switch to the `hsm-v1` suite and
+users and hosts could begin to use new key algorithms while your CAs continue to
+use their existing RSA keys, but whenever a new CA key is generated it may fail
+if the CA uses non-RSA keys.
+New CA keys may be generated if a CA rotation is initiated, or during a version
+upgrade when a new CA has been added in the new version.
+
+### My YubiHSM2 authentication key does not have capabilites for ECDSA keys, what do I do?
+
+Our [YubiHSM2 guide](../../admin-guides/deploy-a-cluster/hsm.mdx) recommends
+creating an "authentication key" to be used by the Teleport Auth server to
```

Comment:
Why is "authentication key" in quotes here?

Reply:
I switched this to a link to the YubiHSM docs